pkgs/development/ocaml-modules/elpi/default.nix

```
@@ -1,13 +1,16 @@
 { lib
 , buildDunePackage, camlp5
+, ocaml
+, menhir, menhirLib
 , re, perl, ncurses
 , ppxlib, ppx_deriving
 , ppxlib_0_15, ppx_deriving_0_15
 , coqPackages
-, version ? "1.14.1"
+, version ? if lib.versionAtLeast ocaml.version "4.07" then "1.15.0" else "1.14.1"
 }:
 with lib;
 let fetched = coqPackages.metaFetch ({
+    release."1.15.0".sha256 = "sha256:1ngdc41sgyzyz3i3lkzjhnj66gza5h912virkh077dyv17ysb6ar";
```

```suggestion
    release."1.15.2".sha256 = "sha256-XgopNP83POFbMNyl2D+gY1rmqGg03o++Ngv3zJfCn2s=";
    release."1.15.0".sha256 = "sha256:1ngdc41sgyzyz3i3lkzjhnj66gza5h912virkh077dyv17ysb6ar";
```
needed to test the coq-master branch of coq-elpi in CIs